repo_nginx_fancyindex/footer.html

```diff
@@ -38,19 +30,19 @@ <h2><span>Post</span>-installation</h2>
 
       <p>Une fois la Brique localisée, connectez-vous par SSH sur le port 22 :</p>
 
-      <pre>root:olinux</pre>
+      <pre>root:yunohost</pre>
```

Comment:
admin?

Reply:
Hm nope, not before the post-install has been ran ... but granted that this can bring confusion :/ Yet root login is always possible on local-network though